protocols/gossipsub/src/config.rs

```diff
@@ -782,6 +805,26 @@ impl ConfigBuilder {
         self
     }
 
+    /// The max number of messages a `ConnectionHandler` can buffer. The default is 5000.
+    pub fn connection_handler_queue_len(&mut self, len: usize) -> &mut Self {
```

Comment:
this setter has the same name as the getter. generally the `with_` prefix is used for setters in the code base
```suggestion
    pub fn with_connection_handler_queue_len(&mut self, len: usize) -> &mut Self {
```

Reply:
yeah, I am aware the thing is that `gossipsub` doesn't follow that logic, I'd suggest we do that for all the methods on another PR wdyt?